src/viper/trans.ml

```diff
@@ -611,6 +621,7 @@ and pat_match ctxt (scrut : M.exp) (p : M.pat) : exp list * (id * T.typ) list *
     end in
     let stmts = List.mapi (fun i (x, t) -> fld_assign_stmt i (LValueUninitVar x)) p_scope in
     [cond], p_scope, (ds, stmts)
+  | M.WildP -> [ !!(BoolLitE true) ], [], ([], [])
```

Comment:
Why add a condition that always holds, i.e. `!!(BoolLitE true)`, when you could simply return an empty list of conditions `[]`?

Reply:
Fair enough, stripped that